### ISSUE TITLE
Catalog problems

### ISSUE THREAD

**Issue by InstallGentoo11** [Pashe/8chanX#26]
http://a.pomf.se/iloydb.webm

I guess after the update of how the catalog works it kind of fucks up with the 8chanX. If you start loading a catalog things will become unclickable until it loads the whole catalog, which has been taking a while in the past few weeks.

I am using firefox 38.0.1 and the pure version of 8chanX.

**Comment by InstallGentoo11** [comment]
Huh, strange. I was trying already a few days ago to find what option from 8chanX could be causing this and I am pretty sure I disabled everything and it was still happening, but it seems like the page indicator was actually the thing doing the problem. Right now loading without it doesn't have that problem. So have that in might, if you are still alive.

**Comment by Pashe** [comment]
Did you try with 8chanX disabled completely?

**Comment by InstallGentoo11** [comment]
Yes, it works fine without 8chanX. As I said the problem seems to be the page indicator. Also as a bonus info I found it is maybe caused only when using https.

**Comment by InstallGentoo11** [comment]
Considering the logjam shit and how the site loads perfectly fine with 8chanX and the indicator on too after I "fixed" my browser.

The problem is now not noticeable because the page loads too fast, but for the moment when moving from small to large pictures in the catalog I think you still can't click, which you can without 8chanX.
